src/components/Profile-card/ProfileCard.style.js

```diff
@@ -19,6 +19,9 @@ export const ProfileCardWrapper = styled.div`
         filter: ${props => props.status === "Inactive" ? "grayscale(1)" : ""};
         top: 50%;
         transition: 0.3s;
+        min-height: 235px;
+        max-height: 235px;
```

Comment:
@Akshit42-hue Giving a fixed height won't work here because if a name has many words then this issue will happen again, try making it by dynamic calculation or truncation after tooo long text (though we don't want to cut any names). You can give a try at both of the solutions or try a new solution.

Reply:
Sounds good :+1:

Reply:
@Chadha93 Can you please merge this PR

Reply:
If you are giving the same value to both `max/min-height` then just replacing it `height` property would do.

Reply:
@Nikhil-Ladha Please do have look